frontend/EmpUI/src/pages/users/details.tsx

```
@@ -226,337 +231,421 @@
+                  <Badge.Ribbon
+                    key={project.projectId}
+                    text={
+                      isProjectActive(project.endDate) ? "Active" : "Completed"
+                    }
+                    color={isProjectActive(project.endDate) ? "green" : "blue"}
+                  >
+                    <Card
+                      hoverable
+                      style={{ width: "100%" }}
+                      actions={[
+                        <Tooltip title="Edit Project">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)